app/models/manageiq/providers/vmware/cloud_manager/vm.rb

```diff
@@ -10,7 +10,7 @@ class ManageIQ::Providers::Vmware::CloudManager::Vm < ManageIQ::Providers::Cloud
   supports :revert_to_snapshot
   supports :reconfigure_disks
   supports :reconfigure_disksize do
-    unsupported_reason_add(:reconfigure_disksize, 'Cannot resize disks of a VM with snapshots') unless snapshots.empty?
+    'Cannot resize disks of a VM with snapshots' unless snapshots.empty?
```

Comment:
Should this have gettext wrappers?
```suggestion
    _("Cannot resize disks of a VM with snapshots") unless snapshots.empty?
```

Reply:
I am unsure if we want to add gettext to all of these.
(also feel like we may want to go towards default values here)